components/inputs/test/input-date-time-range.visual-diff.js

```diff
@@ -280,32 +280,18 @@ describe('d2l-input-date-time-range', () => {
 					});
 
 					it('focus start', async function() {
-						let rect;
-						if (testCase.startDateTooltip) {
-							await focusOnInput(page, '#min-max', startDateSelector);
-							rect = await getRectInnerTooltip(page, '#min-max', startDateSelector);
-						} else {
-							await page.$eval('#min-max', (elem, inputSelector) => {
-								const input = elem.shadowRoot.querySelector(inputSelector);
-								input.focus();
-							}, startDateSelector);
-							rect = await visualDiff.getRect(page, '#min-max');
-						}
+						await focusOnInput(page, '#min-max', startDateSelector);
```

Comment:
The info tooltip now shows up on focus. I can't remember why it wouldn't have before - I'm guessing this is a timing thing since I believe there is a bit of a delay before the tooltip appears. It seems fine to me and actually good that this is being tested.